modules/project_cleanup/function_source/main.go

```diff
@@ -266,8 +298,12 @@ func invoke(ctx context.Context) {
 		localFolderId := strings.Replace(folderId, "folders/", "", 1)
 		logger.Printf("Try to get projects from folder with id [%s] and process them", localFolderId)
 		requestFilter := fmt.Sprintf("parent.type:folder parent.id:%s", localFolderId)
-		req := cloudResourceManagerService.Projects.List().Filter(requestFilter)
-		if err := req.Pages(ctx, processProjectsResponsePage(removeProjectWithLiens)); err != nil {
+		err := retry(func() (err error) {
+			req := cloudResourceManagerService.Projects.List().Filter(requestFilter)
+			err = req.Pages(ctx, processProjectsResponsePage(removeProjectWithLiens))
+			return
+		}, 5, time.Minute)
```

Comment:
5 minutes seems pretty high to start.

Reply:
thats actually the number of times it will retry. Duration is starting with initial 1 minute and 2 * initial for each subsequent try

Reply:
Ack